packages/iov-bns/src/index.ts

```diff
@@ -53,12 +53,19 @@ export {
   // Proposals
   ProposalAction,
   CreateTextResolutionAction,
+  isCreateTextResolution,
   ExecuteProposalBatchAction,
+  isExecuteProposalBatch,
   ReleaseEscrowAction,
+  isReleaseEscrow,
   SendAction,
+  isSend,
```

Comment:
How do you feel about adding the `Action` suffix to them, in order to consistently use the pattern `is<TypeName>` for type checkers?